frontend/values.py

```diff
@@ -32,6 +32,15 @@
     "do podjęcia nauki w szkole"
 ]
 
+sorts = [
```

Comment:
I believe that as this list is limited and will not change, we should instead of `TextField` use `IntegerField` with `choice` attribute set like:
```
IntegerField(choices=((1, 'przedszkole'), (2, 'szkoła podstawowa'), (3, 'technikum'), (4, 'liceum ogólnokształcące'), (5, 'zasadnicza szkoła zawodowa'), (6, 'inne')))
```
More, this `tuple` of `tuples` should be add to `School` model, that instead of
```
from frontend.values import sort
```
you will do
```
from backend.database import School
School.SORTS
```

Reply:
Also, as docs says, nothing is validating this field (see: http://docs.peewee-orm.com/en/latest/peewee/models.html#some-fields-take-special-parameters), so if we want to validate every time we save a model, we can add to this model something like this:
```
def save(self):
    self.validate()
    return super().save()

def validate(self):
    <do some validation>
```

Reply:
I have to admit that I don't know how to get list of str ie. 
`['przedszkole', 'szkoła podstawowa'...]` 
right from model'a choices. Method You suggest (`School.SORTS`) doesn't work. I can do that using list comprehension 
`[x[1] for x in School.sort.choices]` but I feel it's not a good way...